### PR TITLE
test for ClientHello with MD5 and bogus numbers

### PR DESCRIPTION
**Pull request by ep69**

### CODE REVIEW

**Comment by tomato42** [comment]
please add the script to https://github.com/tomato42/tlsfuzzer/blob/master/tests/scripts_retention.py

File: scripts/test-clienthello-md5.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    node = node.add_child(ExpectServerKeyExchange())
+    node = node.add_child(ExpectServerHelloDone())
+    node = node.add_child(ClientKeyExchangeGenerator())
+    node = node.add_child(ChangeCipherSpecGenerator())
+    node = node.add_child(FinishedGenerator())
+    node = node.add_child(ExpectChangeCipherSpec())
+    node = node.add_child(ExpectFinished())
+    node = node.add_child(ApplicationDataGenerator(b"GET / HTTP/1.0\n\n"))
+    node = node.add_child(ExpectApplicationData())
+    node = node.add_child(ExpectClose())
```
Comment (tomato42) [inline scripts/test-clienthello-md5.py]: clean connection close requires sending `connection_close` warning alert and expecting one in return, without this, the test case is essentially time-outing the connection on server side (applies to all test cases)

**Comment by ep69** [comment]
Thanks for review, all the items should be addressed now.

**Comment by tomato42** [comment]
one is still not addressed: https://github.com/tomato42/tlsfuzzer/pull/103#discussion_r98474794 and because of it, Travis run is failing. Also, please squash all changes together and rebase on top of master.

File: tests/scripts_retention.py [hunk 2]
```diff
@@ -157,6 +157,7 @@ def run_rsa_cert_tests(server_cmd):
                       'test-fuzzed-MAC.py',
                       'test-fuzzed-padding.py',
                       'test-hello-request-by-client.py',
+                      'test-clienthello-md5.py',
```
Comment (tomato42) [inline tests/scripts_retention.py]: the names are alphabetically sorted, please put it higher in the list

**Comment by tomato42** [comment]
thank you!

**Comment by ep69** [comment]
Thank you for reviews and patience!

**Pull request merged**